keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -858,6 +853,34 @@ in back-to-back releases.
 - Deprecate the flag
 -->
 
+#### Alpha
+
+- Initial implementation added
+- CRI implementations add support
+
+#### Beta
+
+- Add support for [`subPath`](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath):
+    - By adding a new CRI string `image_sub_path` field to the
+      [`Mount`](https://github.com/kubernetes/cri-api/blob/010fdf8/pkg/apis/runtime/v1/api.proto#L221)
```

Comment:
prefer using an additional image ref only field vs overloading host path for a new purpose